README.md

```diff
@@ -94,18 +112,18 @@ git commits and tags, and push the `.gem` file to
 
 ## Documentation
 
-#### Documentation is Automatically Generated
+### Documentation is Automatically Generated
 
 We maintain API documentation using [YARD](https://yardoc.org/), which is
 published automatically at
-[RubyDoc.info](https://rubydoc.info/github/panorama-ed/memo_wise/main). To edit
-documentation locally and see it rendered in your browser, run:
+[RubyDoc.info](https://rubydoc.info/github/panorama-ed/memo_wise/MemoWise). To
```

Comment:
@ms-ati I'm interested in your thoughts here. My thinking is that the old link just takes people to what appears to be the README again, while this new link takes people to more useful documentation. Thoughts?

Reply:
Great idea!